week-4/D-methods/exercise-4.js

```diff
@@ -3,17 +3,20 @@ Alice has a list of good friends.
 Define a method "makeFriend" to add a new friend to her list.
 */
 
-
 var person = {
-    name: "Alice",
-    friends: ["John", "Nina"]
+  name: "Alice",
+  friends: ["John", "Nina"],
+  makeFriend: function(friend) {
+    this.friends = this.friends + "," + friend;
+  }
```

Comment:
There's a bug here.  person.friends starts as an *array* of strings and you should try to keep it that way. 

For example, it should be possible to say `person.friends.join('---')`.  But if you try to add that line of code to the end of your program you'll find that you cannot.  Something has happened to your `person.friends` property.  

Can you fix your bug?